internal/metastore/table/collection.go

```diff
@@ -0,0 +1,101 @@
+package table
+
+import (
+	"encoding/json"
+	"time"
+
+	"github.com/milvus-io/milvus/internal/common"
+	"github.com/milvus-io/milvus/internal/log"
+	"github.com/milvus-io/milvus/internal/metastore/model"
+	"github.com/milvus-io/milvus/internal/proto/commonpb"
+	"github.com/milvus-io/milvus/internal/util/typeutil"
+	"go.uber.org/zap"
+)
+
+type Collection struct {
+	ID               int64              `db:"id"`
+	TenantID         *string            `db:"tenant_id"`
+	CollectionID     int64              `db:"collection_id"`
+	CollectionName   string             `db:"collection_name"`
+	Description      *string            `db:"description"`
+	AutoID           bool               `db:"auto_id"`
+	ShardsNum        int32              `db:"num_shards"`
+	StartPosition    string             `db:"start_position"`
+	ConsistencyLevel int32              `db:"consistency_level"`
+	Ts               typeutil.Timestamp `db:"ts"`
+	IsDeleted        bool               `db:"is_deleted"`
+	CreatedAt        time.Time          `db:"created_at"`
+	UpdatedAt        time.Time          `db:"updated_at"`
+}
+
+// model <---> db
+
+func ConvertCollectionDBToModel(coll *Collection, collChannel *CollectionChannel, partition *Partition, field *Field, index *Index) *model.Collection {
+	var retDescription string
+	if coll.Description != nil {
+		retDescription = *coll.Description
+	}
+
+	var retFields []*model.Field
+	retFields = append(retFields, ConvertFieldDBToModel(field))
```

Comment:
it is so wired, that it is enough for creating a Field array  with a init value.

Reply:
ok, will change to create an array with init value